frontend/vuepress/config.js

```diff
@@ -55,12 +54,12 @@ module.exports = {
         logo: '/assets/img/victron-logo.png',
 
         // Header navigation items
-        nav: [
-            {
-                text: 'www.victronenergy.com',
-                link: 'https://victronenergy.com',
-            },
-        ],
+        // nav: [
+        //    {
+        //        text: 'www.victronenergy.com',
+        //        link: 'https://victronenergy.com',
+        //    },
+        // ],
```

Comment:
Let's not comment out things we mean to remove. Preserving history is what we have Git for. :)